contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -567,7 +673,7 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
     for (uint256 idx = 4; idx < 15; idx++) {
       if (rawID[idx] != empty) {
         // old IDs that were created before this standard and migrated to this registry
-        return Trigger.CONDITION;
+        return Trigger.BLOCK;
```

Comment:
imo conditional represents the trigger much better than block, which signifies that it will be done on every block which is not the case

Reply:
changed it to better encapsulate conditional and "run when ready" types